src/style.css

```
@@ -0,0 +1,69 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.blocks__menu {
+  display: flex;
+  flex-wrap: wrap;
+  color: #fff;
+}
+
+.blocks {
+  height: 300px;
+  min-width: 300px;
+  text-align: center;
+  line-height: 300px;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  flex-grow: 1;
+  flex-basis: 33%;
+}
+
+.block1 {
```

Add some dashes, to make modificator stand out 
```suggestion
.block--1 {
```